app/controllers/application_controller.rb

```diff
@@ -12,6 +12,19 @@ class ApplicationController < ActionController::Base
 
 protected
 
+  def get_payload(payload = nil)
+    if payload
+      jwt = Jwt.create!(
+        jwt_payload: payload,
+      )
+      session[:jwt_id] = jwt.id
+      payload
+    elsif session[:jwt_id]
+      jwt = Jwt.find(session[:jwt_id])
+      jwt.jwt_payload.deep_symbolize_keys
```

Comment:
I think this should be `jwt&.jwt_payload.deep_symbolize_keys`, so if a user somehow ends up with the ID of a expired JWT in their session they don't get a 500 error.